blueprints/hr_assistant/hierarchy.py

```diff
@@ -15,7 +15,12 @@ def get_hierarchy_up(request, responder):
     """
 
     try:
-        name_ent = [e['value'][0]['cname'] for e in request.entities if e['type'] == 'name']
+
+        if request.frame.get('info_visited'):
+            name = request.frame.get('name')
```

Comment:
Do we want to check for null and handle it here if the frame does not have the key?